### PR TITLE
#8 add myself to Contributors.md file

### PR DESCRIPTION
**Pull request by ACIDKing**
#8

### CODE REVIEW

**Comment by arshadkazmi42** [comment]
@ACIDKing Looks good. Can you tag the issue to pull request description. I will merge it as soon as that is done.

Also, always work on a feature branch, its not good practice working on master branch

**Comment by ACIDKing** [comment]
@arshadkazmi42 Done! Thanks, I will be more attentive next time

**Comment by arshadkazmi42** [comment]
@ACIDKing You have to tag it in the description not in the title.

**Comment by ACIDKing** [comment]
@ACIDKing Done

File: Contributors.md [hunk 1]
```diff
@@ -8,3 +8,5 @@
 - [Shubham Kumar](https://github.com/sk9331657)
 - [Ipshita Chatterjee](https://github.com/IpshitaC)
 - [Danil Shankovskiy](https://github.com/BeAsYit)
+- [Viacheslav_Ignatov](https://github.com/ACIDKing)
+
```
Comment (arshadkazmi42) [inline Contributors.md]: can you remove this empty space

**Comment by arshadkazmi42** [comment]
congrats on landing your first PR.

**Pull request merged**